consensus/src/chained_bft/test_utils/mock_storage.rs

```diff
@@ -273,7 +278,12 @@ impl<T: Payload> PersistentLivenessStorage<T> for EmptyStorage<T> {
             }
         }
     }
+
     fn save_highest_timeout_cert(&self, _: TimeoutCertificate) -> Result<()> {
         Ok(())
     }
+
+    fn libra_db(&self) -> Arc<dyn LibraDBTrait> {
```

Comment:
would recommend making a wrapper around LibraDB called SharedLibraDB(Arc<LibraDB>) that re-exposes the LibraDBTrait, but maybe another time another place....